packages/design-system/src/components/__tests__/smoke.e2e.test.js

```diff
@@ -1,22 +0,0 @@
-/* global driver */
-import { getElementById, getElementByXPath } from '@cmsgov/design-system-scripts/helpers/e2e';
```

Comment:
@pwolfert Moving our Slack conversation to the PR...
I have deleted this file instead of trying to support one singular e2e test file. My reasoning is that if you run `yarn test:a11y`, as part of the 'global-setup' file, `yarn build` is run. If the doc site generation fails (which is what this test is testing), won't it get caught in that setup file?

Reply:
Yeah, that's true. This tests that generating all the example pages works. What's not necessarily tested is whether the docs site pages are generated properly, but if we really want that coverage, we can come back later and address it. It's not often that we break the docs site without knowing it. When it happened last week, the whole build command failed, so the `a11y` tests certainly do cover that casse 👍